test/rpc-server/implies/not-implied-stuck/README.md

```diff
@@ -0,0 +1 @@
+`a => b`, response `False`, with substitution `[b/a]`
```

Comment:
Why is this stuck? Also, we usually prefer variables to be have names starting with an upper-case letter (like enforced in K). I initially thought `a` and `b` were symbols (specifically, constructors) until I saw the JSON.

Reply:
I first thought this fails because the `b` is all-quantified (instead of existentially) but thinking about it again, you are right, this should actually _pass_. Maybe another case similar to #3218 ? Will rework the test to use a different example for now.

Reply:
We should add a check which makes sure that any extra RHS variables are existentially quantified.